src/topology/uniform_space/uniform_convergence.lean

```diff
@@ -55,21 +62,118 @@ open_locale topological_space classical uniformity filter
 
 open set filter
 
+section generic
+
+variables {α β γ ι : Type*} {p : filter ι} {p' : filter α}
+
+lemma filter.eventually_swap_iff {f : (ι × α) → Prop} : (∀ᶠ (x : ι × α) in (p.prod p'), f x) ↔
+  ∀ᶠ (y : α × ι) in (p'.prod p), f y.swap :=
```

Comment:
I had the above as pending review comment last time. I'll do a full review after you clean this PR since the other PR was merged (I will be a lot more responsive from now on).

Reply:
I migrated this in the previous commit to filter.prod, so this is already resolved :-)